<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ipV4calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QLineEdit" name="leIPv4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000.000.000.000</string>
    </property>
    <property name="text">
     <string>93.185.28.238</string>
    </property>
   </widget>
   <widget class="QLabel" name="lIPv4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input IPv4 address</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNetmask">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input netmask</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCompute">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teResults">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lResults">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbNetmask">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
